<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseEditor</class>
 <widget class="QMainWindow" name="DatabaseEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="databases">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Local database</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="songs">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewSong"/>
   <addaction name="actionEditSong"/>
   <addaction name="actionDeleteSongs"/>
   <addaction name="separator"/>
   <addaction name="actionNew_songgook"/>
   <addaction name="separator"/>
   <addaction name="actionReload"/>
  </widget>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEditSong">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit song</string>
   </property>
  </action>
  <action name="actionNewSong">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New song</string>
   </property>
  </action>
  <action name="actionDeleteSongs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete song(s)</string>
   </property>
  </action>
  <action name="actionNew_songgook">
   <property name="text">
    <string>New songbook</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DatabaseEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
